numpy/core/fromnumeric.py

```diff
@@ -62,6 +61,32 @@ def _wrapfunc(obj, method, *args, **kwds):
         return _wrapit(obj, method, *args, **kwds)
 
 
+def _wrapreduction(obj, method, name, axis, dtype, out, **kwargs):
+    passkwargs = {}
+    for k, v in kwargs.items():
```

Comment:
I would just stick with `keepdims` at the top and keep the test as is. This is only meant as a work-around for old code that does not support `keepdims`; if we add other keywords, we can just expand.

Reply:
I just thought this makes it easy to add kwargs to reductions, like `initializer` in my other PR.

Reply:
I hadn't seen that PR when I reviewed...